x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/utils.ts

```diff
@@ -33,16 +34,18 @@ export const createSlackConnectorAndObjectRemover = async ({
 export const createSlackConnector = async ({
   name,
   supertest,
+  getService,
 }: {
   name: string;
   supertest: SuperTest.SuperTest<SuperTest.Test>;
+  getService: FtrProviderContext['getService'];
```

Comment:
We can remove line 40 since `supertest` isn't used anymore.

Reply:
Fixed in 701ef860c89369cdbf86cc5ebc6b158fb5e2c7d2